src/block/block_mgr.c

```diff
@@ -631,13 +631,11 @@ __wt_block_manager_open(WT_SESSION_IMPL *session, const char *filename, const ch
 
 /*
  * __wt_block_panic --
- *     Report an error, then panic the handle and the system.
+ *     Set the block API to read-only.
  */
-int
+void
 __wt_block_panic(WT_SESSION_IMPL *session) WT_GCC_FUNC_ATTRIBUTE((cold))
```

Comment:
This is now a poorly named function. Do we need it, given that it is only a single line? Can it be a macro instead?

Reply:
I'd like to keep it as a function for information-hiding reasons. I've renamed it to `__wt_block_set_readonly()`, I agree the name no longer makes sense.

Pushed in 4636a3aaf.